src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCNewModifyPage.java

```diff
@@ -125,31 +124,47 @@ public void setContactPhoneTextField(final String contactPhoneTextField) {
 
     /**
      * Method to set contact las Name attribute.
+     *
      * @param contactHomePhoneTextField string value.
      */
     public void setContactHomePhoneTextField(final String contactHomePhoneTextField) {
         this.contactHomePhoneTextField.clear();
         this.contactHomePhoneTextField.sendKeys(contactHomePhoneTextField);
     }
+
     /**
      * Method to do click "save" button.
      */
     public void clickSaveContactButton() {
         getSaveContactButton().click();
     }
+
+
     /**
-     * Method to do click "fillMethodsToFields" to map.
+     * @param values contact field
      * @return map of contact fields.
      */
-    public Map<String, Value> fillMethodsToFields() {
-        newContactsFields.put("contactName", this::setContactNameTextField);
-        newContactsFields.put("contactLastName", this::setContactLastNameTextField);
-        newContactsFields.put("contactTitle", this::setContactTitleTextField);
-        newContactsFields.put("contactDepartment", this::setContactDepartmentTextField);
-        newContactsFields.put("contactBirthday", this::setContactBirthdateTextField);
-        newContactsFields.put("contactLeadSource", this::setContactLeadSourceTextField);
-        newContactsFields.put("contactPhone", this::setContactPhoneTextField);
-        newContactsFields.put("contactHomePhone", this::setContactHomePhoneTextField);
-        return newContactsFields;
+    public Map<Inputs, Value> getStrategyStepMap(final Map<Inputs, String> values) {
+        Map<Inputs, Value> strategyMap = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)